src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,113 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
+@SuppressWarnings("unchecked")
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private int capacity;
+    private T[] arrayList;
+    private int size;
```

Comment:
```suggestion
    private T[] values;
    private int size;
```
Lets operate with size value, without capacity